src/main/pages/che-7/end-user-guide/proc_starting-a-workspace-with-a-devfile.adoc

```diff
@@ -1,27 +1,63 @@
 [id="starting-a-workspace-with-a-devfile_{context}"]
-= Starting a workspace with a Devfile
+= Starting a workspace with a devfile
+
+This section describes how to start a Che workspace using your own existing devfile.
 
-This paragraph is the procedure module introduction: a short description of the procedure.
 
-[discrete]
 .Prerequisites
+ 
+* A running instance of Eclipse Che.
+* An existing devfile.
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
 
-[discrete]
 .Procedure
 
-. Start each step with an active verb.
+There are several ways to launch a Che workspace based on a prepared devfile:
+
+.Devfile from Git
+
+Deploy the devfile to a Git source repository and then create a factory from this repository.
+
+. Put a `devfile.yaml` file in the root of your Git repository.
+. Execute the factory by opening the following URL:
++
+[subs="+quotes"]
+----
+https://__<your-che-host>__/f?url=https://github.com/__<mygroup>__/__<myrepo>__
+----
+
+.Devfile from URL
+
+Execute a devfile by constructing the factory with the URL pointing to the raw content of the devfile, for example:
+
+[subs="+quotes"]
+----
+https://__<your-che-host>__/f?url=https://pastebin.com/raw/ux6iCGaW
+----
+
+.Devfile over the Che REST API
+
+Send a devfile to the dedicated REST API using `curl` or Swagger, which will create a new workspace and return its configuration:
+
+[subs="+quotes"]
+----
+$ curl -X POST  -H "Authorization: __<TOKEN>__" -H "Content-Type: application/yaml" -d _<devfile_content>_ https://__<your-che-host>__/api/devfile
+----
+
+.Devfile using the `chectl` tool
+
+Execute workspace from a devfile using the `workspace:start` parameter with the `chectl` tool as follows:
 
-. Include one command or action per step.
+[subs="+quotes"]
+----
+$ chectl workspace:start --devfile=devfile.yaml
+----
 
-. Use an unnumbered bullet (*) if the procedure includes only one step.
+Note that this way only works with local (same machine) devfiles -- URLs cannot be used with `chectl`.
```

Comment:
This would mean to me that the same restriction as with the devfiles posted to the REST endpoint applies here, too, but confirmation from the chectl folks would be needed.

Reply:
Removed as per Mario's comment; it now works with remote.